src/vue/backend/src/views/children/advancedContentTab.vue

```diff
@@ -24,8 +25,14 @@ export default {
     components: {
         impressListingsAdvancedContent
     },
+    data () {
+        return {
+            formDisabled: true
+        }
+    },
     methods: {
         async saveHandler () {
+            this.formDisabled = true
             const { status } = await this[repo].post(this.formChanges, 'advanced')
```

Comment:
Once this loads, you can set form disabled back to false
*Repeat for all instances